### PR TITLE
Make a perpetually open bounty for approved GitHub issues

### PR DESCRIPTION
**Pull request by JST5000**
This way as issues are found which would make for good bounties, we can easily add them to this open bounty.

### CODE REVIEW

File: open_bounties/0000 Xrpl.js Open Bounties.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+If an open issue in xrpl.js has the tag "Approved Bounty", it is eligible to be completed for the specified dollar amount.
+These tasks generally take at least a day of coding, are reasonable to build on your own, and are a way to learn about
+a specific piece of the XRPL ecosystem.
+
+In order to complete the bounty, your PR must include unit tests, be well documented, and be merged in to the xrpl.js repository.
```
Comment (jonathanlei) [inline open_bounties/0000 Xrpl.js Open Bounties.md:33]: edit changlog.md

Comment (JST5000) [reply open_bounties/0000 Xrpl.js Open Bounties.md]: > I think in addition to labeling issues as "Open Bounties", we still need to list them here, each with requirements (if clarification is needed in addition to the issue) and award amount.

Agreed. We'll need a couple examples to do that though, so will likely have to wait until we've finished grooming past issues for good bounties.

**Comment by JST5000** [comment]
> I think in addition to labeling issues as "Open Bounties", we still need to list them here, each with requirements (if clarification is needed in addition to the issue) and award amount.

Agreed. We'll need a couple examples to do that though, so will likely have to wait until we've finished grooming past issues for good bounties.

File: open_bounties/0000 Core Library Support.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+-->
+
+For core XRPL libraries such as xrpl.js, xrpl-py, rippled, and the explorer, there may be issues which make for good bounties.
+
+Those issues should also be tagged with "Approved Bounty" once they are listed here.
+
+These tasks generally take at least a day of coding, are reasonable to build on your own, and are a way to learn about
+a specific piece of the XRPL ecosystem.
+
+In order to complete the bounty, your PR must include unit tests, be well documented, update the CHANGELOG.md / HISTORY.md file and be merged in to the corresponding repository.
```
Comment (khancode) [inline open_bounties/0000 Core Library Support.md:36]:
```suggestion
In order to complete the bounty, your PR must include tests (unit and/or integration), be well documented, update the CHANGELOG.md / HISTORY.md file and be merged into the corresponding repository.
```

File: open_bounties/0000 Core Library Support.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+This is a purpetual bounty - so as new issues are discovered we will continue to add milestones here.
```
Comment (khancode) [inline open_bounties/0000 Core Library Support.md:38]:
```suggestion
This is a perpetual bounty - so as new issues are discovered we will continue to add milestones here.
```

Correct spelling is perpetual 🤓

**Review by khancode — CHANGES_REQUESTED** [review]

**Review by intelliot — COMMENTED** [review]
I actually think it would be preferable to just open specific bounties for important GitHub issues, instead of having a "perpetual" one. There should only be a few such bounties per month, so it should be manageable. 

In any case, the bigger problem is that there aren't any approved bounties still open right now, right?

https://github.com/XRPLF/xrpl.js/labels/approved%20bounty

**Pull request closed without merging**